ios/Example/Podfile

```diff
@@ -10,3 +10,9 @@ target 'Trifle_Example' do
 
   end
 end
+
+target 'Trifle_macOS_Dummy' do
```

Comment:
dummy target that is macOS based is created in order to pull down the wire compiler to be used for generating wire format as specified in the readme